packages/flutter/lib/src/material/stepper.dart

```diff
@@ -360,28 +389,31 @@ class _StepperState extends State<Stepper> with TickerProviderStateMixin {
   }
 
   Widget _buildCircleChild(int index, bool oldState) {
-    final StepState state = oldState ? _oldStates[index]! : widget.steps[index].state;
-    final bool isDarkActive = _isDark() && widget.steps[index].isActive;
```

Comment:
Here we seem to be removing old behavior which automatically adapted the [Stepper] to a dark theme. Am I understanding this change correctly? If so, I would think those dark theme values are still important.